pages/invites.tsx

```diff
@@ -0,0 +1,127 @@
+//  import DashboardLayout from "components/DashboardLayout";
```

Comment:
let's make this into a component since this is 1 of the 3 things included in our invites table. also since this feature is a dashboard for user requests, we should call this component something like userRequests